crates/sui-indexer-alt-graphql/src/api/types/transaction_kind/change_epoch.rs

```diff
@@ -63,4 +73,50 @@ impl ChangeEpochTransaction {
             self.native.epoch_start_timestamp_ms as i64,
         )?))
     }
+
+    /// System packages (specifically framework and move stdlib) that are written before the new epoch starts, to upgrade them on-chain. Validators write these packages out when running the transaction.
```

Comment:
nit: there are other system packages other than the framework and stdlib (the `sui_system` package for governance, deepbook, the bridge).

```suggestion
    /// System packages that will be written by validators before the new epoch starts, to upgrade them on-chain. These objects do not have a "previous transaction" because they are not written on-chain yet. Consult `effects.objectChanges` for this transaction to see the actual objects written. 
```

Reply:
SG. Updated